mandatory/4-tax.js

```diff
@@ -5,7 +5,13 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(productPrice) {
+  const salesTax = (productPrice / 100) * 20;
```

Comment:
Where you have numbers that don't inherently mean anything e.g. `20` here, it's often good to assign them to a variable to make their meaning clear in the code e.g.:

```
const SALES_TAX_RATE = 20;
const salesTax = (productPrice / 100) * SALES_TAX_RATE;
```

If you're wondering why the caps, you'll often see constants expressed like this in JavaScript